packages/neo-one-website/src/components/common/SidebarLink.tsx

```diff
@@ -0,0 +1,42 @@
+import { Link } from '@neo-one/react-common';
+import * as React from 'react';
+import { Link as RouterLink } from 'react-static';
+import { as, Button, Hidden, styled } from 'reakit';
+import { prop } from 'styled-tools';
+import { HiddenAPI } from './types';
+
+export interface Props {
+  readonly path: string;
+  readonly title: string;
+  readonly hidden?: HiddenAPI;
+}
+
+const StyledLink = styled(as(RouterLink)(Link))`
+  &:focus {
```

Comment:
This should probably just be on the base link I would think. Seems like all links should change color when they're focused (e.g. via tabbing), so let's move it there.